### PR TITLE
feat(proxy-12): added validation to endpoints

### PR DESCRIPTION
**Pull request by denStepaniuk**

### CODE REVIEW

File: delivery/router/errors/error-handler.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 const {customError} = require("./errors");
```
Comment (plutsenka) [inline delivery/router/errors/error-handler.js]: Please check usages of quotation marks - it's better to use them in one style - only '' or only "" - in all project

File: delivery/router/router.js [hunk 2]
```diff
@@ -13,18 +18,30 @@ router.use(express.json());
 router.get('/meteors', (req, res, next) => {
   if (Object.keys(req.query).length !== 0) {
     retrieveMeteorDataWithQueryParams(res, req.query)
+    .then((responseBody) => {
```
Comment (plutsenka) [inline delivery/router/router.js]: Probably this logic can be moved to controller

**Review by plutsenka — APPROVED** [review]

File: delivery/router/errors/error-handler.js (hunk 1 above)
Comment (mariaYurcheuskaya) [reply delivery/router/errors/error-handler.js]: you can use [https://prettier.io/](url) for it)

Comment (denStepaniuk) [reply delivery/router/errors/error-handler.js]: Was adjusted to one approach.

File: delivery/router/router.js (hunk 2 above)
Comment (denStepaniuk) [reply delivery/router/router.js]: Cleaned router and moved out to separated controller layer.

**Pull request merged**